playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -8,3 +8,8 @@
       role: common
       local_facts:
         hostname: "{{ openshift_hostname | default(None) }}"
+  - set_fact:
+      openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+  - set_fact:
+      repoquery_cmd: "{{ 'dnf repoquery --latest-limit 1 -d 0' if ansible_pkg_mgr == 'dnf' else 'repoquery' }}"
```

Comment:
Depending on the future of `openshift_facts` this may need to become a common fact also since we're using it in multiple places rather than setting it here.

Also calling attention to the updates to repoquery commands in https://github.com/openshift/openshift-ansible/pull/2101